src/tokens.js

```diff
@@ -755,16 +763,16 @@ const tokens = {
    *
    * @throws {InsufficientFundsError} If not enough tokens for deposit
    *
-   * @return {Object} Mint inputs/outputs data {'inputs', 'outputs'}
+   * @return {Promise} Promise resolved with object containing mint inputs/outputs data {'inputs', 'outputs'}
    *
    * @memberof Tokens
    * @inner
    */
-  getMintDepositInfo(mintAmount) {
+  async getMintDepositInfo(mintAmount) {
```

Comment:
We also have some functions that are `async` and others are not. Personally, I prefer to use `async` as the code becomes easier to read. Maybe we should open an issue to refactor the lib and only use one style.